internal/pkg/pluginengine/change.go

```diff
@@ -40,27 +40,31 @@ const (
 // GetChangesForApply takes "State Manager" & "Config" then do some calculate and return a Plan.
 // All actions should be execute is included in this Plan.changes.
 func GetChangesForApply(smgr statemanager.Manager, cfg *configloader.Config) ([]*Change, error) {
-	return getChanges(smgr, cfg, CommandApply)
+	return getChanges(smgr, cfg, CommandApply, false)
```

Comment:
if "isForce" needs to be passed to "getChanges", then the name isn't good enough. Because according to the logic of "getChanges", it can get changes for both delete, and apply. What does "isForce" mean exactly? Is apply force? Or is delete force? Or both?

We could have a better naming, like isForceDelete, or simply forceDelete. Or, we can refactor getChanges so that it doesn't depends on forceDelete.